share/sysobject.ids

```diff
@@ -3791,6 +3791,7 @@
 529.1.9	Ascend	NETWORKING	APX-1000
 
 534	Eaton	NETWORKING
+534.6.6.7   Eaton   NETWORKING  Eaton ePDU
```

Comment:
Here you should respect the OIDs order and place your entry after `534.1`.

Reply:
Do i delete this modification from the commits here?

Reply:
No you don't need it will just be overwritten during the next release as I merged you sysobject.ids PR.